feedback/pretalx_client.py

```
@@ -0,0 +1,1050 @@
+            for k, v in params.items():
+                if v is not None:
+                    if isinstance(v, list):
+                        cleaned_params[k] = ",".join(map(str, v))
+                    elif isinstance(v, bool):
+                        cleaned_params[k] = "true" if v else "false"
+                    else:
+                        cleaned_params[k] = str(v)
+            if cleaned_params:
+                url = f"{url}?{urllib.parse.urlencode(cleaned_params)}"
+                
+        req = urllib.request.Request(url, method=method)
```

Ruff (0.16.3)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)